robots_101/programme_structure.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+title: Robots 101 - Programme Structure
+---
+
+# Robots 101 - Programme Structure
+
+Student Robotics is an annual robotics competition for people aged 16-19.
+But what actually happens during the course of the competition year?
+
+## Registration
+
+Registration for the competition usually open in late summer. Teams usually
+consist of 4-8 competitors attending a school or college, though other groups
+are also free to enter. Your team must have a responsible adult who will oversee
+the team (a role we call the [team supervisor][team-supervisor]) and they must
+be the person to [sign up][sign-up].
+
+While places are free (thanks to our [sponsors][sponsors]), they are limited.
+You will be notified whether or not your team has received a place shortly
+before the start of the competition year.
+
+<!-- TODO: add something about the pre-kickstart activities once they're published -->
+
+## Kickstart
+
+Kickstart is the event which kicks off a competition year. The structure of the
+competition and the [game rules][rules] are announced and any questions you have
+will be answered.
+
+Typically this is a one day in-person event in October (usually a Saturday).
+Throughout the day there are Blueshirts (our awesome volunteers!) around in
+person, ready to answer any questions and to help you through the
+[microgames][microgames]. The microgames help you become familiar with the
+[kit][kit] which is loaned to all teams and should kickstart some ideas for your
+robot’s design.
+
+## Mentoring
+
+Shortly after Kickstart we aim to pair up teams with a volunteer mentor who can
+guide the team and help them with any technical questions. Where possible the
+mentor will join the team's meetings in person, though we also offer remote
+mentors if that is not possible.
+
+All teams also have access to our [Discord][discord] server where they can chat
+with all our volunteers (as well as other teams) at any time.
+
+## Your time
+
+Most teams work on their robot for at least two hours a week, with some teams
+spending four hours or more. How this time is scheduled varies between teams --
+some have more than one meeting a week, others have less frequent but longer
+sessions.
+
+To get the most out of the time, ensure sessions are long enough to make useful
+progress after allowing for setting up beforehand and tidying away afterwards.
+
+## Tech Days
+
+Throughout the year [Tech Days][tech-days] provide opportunities to spend a
+whole day (usually a Saturday) working on your robot, with lots of help
+available. They’re also an opportunity to see how other teams are doing or get
+more direct help with your robot.
+
+We provide a space for you to work in, with power and internet access, as well
+as volunteers able to help with the kits and hands-on guidance with your robot.
+
+## Competition
```

Comment:
There's no mention of the virtual competition here - should there be? :)

Reply:
+ friendlies.

Reply:
There's also no mention of the Challenges either. I deliberately didn't mention them -- they felt "small" enough not to worry about as well as being a bit more tied to the specifics of the content of the year rather than the structure. The Virtual Competition perhaps is different though :thinking: I don't think the friendlies are though (plus we don't always run them).

Reply:
Agreed with the challenges and friendlies - may be worth mentioning the virtual comp.

Reply:
30a041a